common/src/main/java/brocoeur/example/common/roulette/RoulettePlay.java

```diff
@@ -3,7 +3,61 @@
 import brocoeur.example.common.GamePlay;
 
 public enum RoulettePlay implements GamePlay {
-    RED,
-    BLACK,
-    GREEN
+    // Colors
+    RED(2, "RED"),
+    BLACK(2, "BLACK"),
+    // Numbers
```

Comment:
Will have to add the plays where we touch multiple cases at the same time and thus divide the multipliers but that also implies localisation factor which makes this even more complex

Reply:
jsut a POC, to be upadted/reviewed later.